### DISCUSSION TITLE
Very long loading: Opening /admin-Page

### DISCUSSION THREAD

**Discussion by Der-K-2000** [YOURLS/YOURLS#3630] [General]
Hi, I'm using YOURLS for a very long time (> 10 years?) and so far for multiple projects. For a new shortlinker-instance everything seemed fine. But since I tried to convert a link to a page which is loading very very long (isn't there a time-out for YOURLS?) everything messed up a bit. Probably the page has nothing to do with it, but it started exactly at that time.

Finally after minutes (!) the link was converted, but since then everytime I open the overview `/admin/index.php` it took also a minute (?) to load that page, while a new link converting (when the page is available) is fast as before. The `/admin/plugins.php` and others are immediately available. Currently only 1 k links are converted. And I haven't seen this issues before.

Any idea, where to start? Unfortunatly this instance it's installed on a simple webhoster, not ony my servers for the other projects, so my ressources to view log files etc. are restricted.

**Comment by dgw** [comment]
First I would make note of the keyword & long URL that appeared to prompt this problem.

Try to filter URLs using the search fields below the table, first with a query that will **not** match the possibly problematic link, then with a query that **will only** match the possibly problematic link.

If the query that does **not** match loads quickly, and the query that **only** matches loads slowly, please share the URL here.

The best workaround could end up being deletion of the problematic shortlink (and that would be my next step if these search-based tests don't show any difference), but non-destructive checks are always best tried first.
